src/Microsoft.Restier.WebApi/Query/QueryTotalCount.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Restier.Core.Query;
+
+namespace Microsoft.Restier.WebApi.Query
+{
+    internal class QueryTotalCount : IQueryExecutor
+    {
```

Comment:
@rayao @chinadragon0515 Conventionally in `Restier.WebApi`, we name things with the `Restier` prefix instead of the `OData` prefix, such as `RestierController`, `RestierPayloadValueConverter`, etc. This is to differentiate from the names in OData Web API and ODataLib. Thus to keep consistent, please follow this convention (e.g., `RestierQueryExecutor` and `RestierQueryExecutorOptions`). Besides I have no other concerns. Thanks!